velox/exec/tests/CMakeLists.txt

```diff
@@ -99,6 +99,8 @@ add_test(
   COMMAND velox_exec_test
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 
```

Comment:
Can we look at each table writer test case to see if there is any long running unit test case and reduce its execution time? And also consider to run multiple iterations on devserver.